scenarios/pubsub_pep.erl

```diff
@@ -60,13 +60,9 @@ start(Id, Settings) ->
     start_user(Client, Settings).
 
 init_metrics() ->
-    Counters = [message,
-                %% node counters
-                node_creation_failure, node_creation_success, node_creation_timeout,
-                %% publication counters
-                publication_query, publication_result, publication_error,
-                publication_success, publication_timeout],
-    Times = [node_creation, publication, message_ttd],
+    iq_metrics:start([node_creation, publication]),
+    Counters = [message],
```

Comment:
The name of a metric can be just an atom like this `message`.

Reply:
Sorry, I'm not sure what you mean. There are two groups of metrics handled by iq helper: `node_creation` and `publication`, which are both atoms. Do you mean that i.e. there should be `publication` metric instead of `publication.request`?

Reply:
As far as I can see the `Counters = [message]` var is used in `amoc_metrics:init`. This function accepts just atom as a name of the function. I thought that it may be a bit more readable if we use just atom as a name of the counter metrics.